command_line/background.py

```diff
@@ -67,90 +69,95 @@ def run(args=None):
 
     # Ensure we have either a data block or an experiment list
     experiments = flatten_experiments(params.input.experiments)
-    if len(experiments) != 1:
+    if 0 and len(experiments) != 1:
```

Comment:
```suggestion
    if not experiments:
```

Reply:
Thanks - I'd removed this code block locally but missed it off my commit